Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM scratch
+
+COPY ./dist/linux-amd64/esevaluator /app
+
+EXPOSE 8080
+
+ENTRYPOINT ["/app"]
```

Comment:
Add a `CMD ["0.0.0.0:8080"]` so the container is listening to all interfaces, otherwise it won't be able to communicate outside of the container.